python/glide-async/python/glide/__init__.py

```diff
@@ -314,15 +261,13 @@
     "SlotKeyRoute",
     "SlotIdRoute",
     "TSingleNodeRoute",
-    # Exceptions
```

Comment:
lets add these sections back, e.g. # Exceptions , # Routes.. also for the sync client